.circleci/config.yml

```diff
@@ -219,6 +219,13 @@ commands:
           branch_pattern: develop
           mentions: "<< parameters.mentions >>"
 
+  install-zstd:
+    description: "Install zstd compression utility"
+    steps:
+      - run:
+          name: Install zstd
+          command: sudo apt-get update && sudo apt-get install -y zstd
```

Comment:
Hmmm this is tricky - local installation is then screwed. I would at least pin the version at this point